src/ui/views/Approval/components/DefaultBlock.tsx

```diff
@@ -29,6 +29,40 @@ export const DefaultBlock = ({
 }) => {
   const history = useHistory();
 
+  const hexToString = (hex) => {
+    let str = '';
+    for (let i = 0; i < hex.length; i += 2) {
+      str += String.fromCharCode(parseInt(hex.substr(i, 2), 16));
+    }
+    return str;
+  };
+
+  const processItem = (item) => {
+    if (Array.isArray(item)) {
+      return `[${item.map((value) => processItem(value)).join('')}]`;
```

Comment:
I think we need "," as separator for array items

Reply:
![image](https://github.com/user-attachments/assets/c269287b-a500-4e0a-b862-68bb07260599)